src/main/java/science/icebreaker/network/NetworkService.java

```diff
@@ -0,0 +1,105 @@
+package science.icebreaker.network;
+
+import org.neo4j.driver.*;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.neo4j.driver.Values.parameters;
+
+
+@Service
+public class NetworkService implements AutoCloseable {
+
+    /**
+     * TODO Find a better solution to limit the size of the results.
```

Comment:
Is there a related issue?

Reply:
I have added a task on the Notion page for the network.